assets/admin/editor-wizard/use-editor-wizard-steps.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * WordPress dependencies
+ */
+import { useSelect } from '@wordpress/data';
+import { store as editorStore } from '@wordpress/editor/build/store';
+
+/**
+ * Internal dependencies
+ */
+import CourseDetailsStep from './steps/course-details-step';
+import CourseUpgradeStep from './steps/course-upgrade-step';
+import CoursePatternsStep from './steps/course-patterns-step';
+import LessonDetailsStep from './steps/lesson-details-step';
+import LessonPatternsStep from './steps/lesson-patterns-step';
+import { EXTENSIONS_STORE } from '../../extensions/store';
+
+const useEditorWizardSteps = () => {
```

Comment:
We could add a JSDoc here explaining this hook purpose.

Reply:
Fixed in [02f8c68](https://github.com/Automattic/sensei/pull/5186/commits/02f8c6809094de81474a1e65b043340e081b83c5). Thanks!